pkg/logql/range_vector.go

```diff
@@ -178,6 +193,7 @@ func (r *batchRangeVectorIterator) load(start, end int64) {
 			series.Metric = metric
 			r.window[lbs] = series
 		}
+		// TODO(twhitney): Everywhere else, an FPoint.T is in milliseconds, but here it's in nanoseconds.
```

Comment:
it's not something I'm going to touch in this already large PR, but yes, I think we can address this one usage. My understanding is that it doesn't affect the result because this is inside a step evaluator, and the step evaluator's `.Next()` returns the timestamp we use in the actual response, which is correct. I just want to fix this for consistency, but, in a separate PR.